### DISCUSSION TITLE
hyphen in username blocks pushing docker image

### DISCUSSION THREAD

**Discussion by mees-** [community/community#40265] [Actions]
### Select Topic Area

Bug

### Body

I can't publish docker images to the github package registry because my username ends with a hyphen.  
Docker won't let me tag the image with my username: `ghcr.io/mees-/repository:latest` gives me the following output:
```
❯ docker tag repository:latest ghcr.io/mees-/repository:latest
Error parsing reference: "ghcr.io/mees-/repository:latest" is not a valid repository/tag: invalid reference format
```

Is there some alternative character that I can use in docker tags that is accepted by github so they still know it's me?  
Or is there some other way, without changing my username, to publish docker images to ghcr?

**Comment by ossanna16** [comment]
Hi there @mees- and welcome to our community! Thank you for reporting this bug!
